Patches/python.diff

```diff
@@ -75,62 +75,40 @@ diff -r 8667c26e2bec Include/dictobject.h
  #ifdef __cplusplus
 diff -r 8667c26e2bec Include/pystate.h
 --- a/Include/pystate.h	Mon Jun 15 09:11:37 2015 -0700
-+++ b/Include/pystate.h	Mon Aug 31 14:44:51 2015 -0700
-@@ -12,6 +12,8 @@
++++ b/Include/pystate.h	Thu Jan 07 11:46:53 2016 -0800
+@@ -12,6 +12,9 @@
  
  struct _ts; /* Forward */
  struct _is; /* Forward */
 +typedef void*(__stdcall *CompileFunction)(PyObject*);
++typedef void*(__stdcall *JitFreeFunction)(PyObject*);
 +typedef void(__stdcall *JitInitFunction)();
  
  #ifdef Py_LIMITED_API
  typedef struct _is PyInterpreterState;
-@@ -41,6 +43,7 @@
+@@ -41,6 +44,8 @@
  #endif
  
      PyObject *builtins_copy;
-+	CompileFunction jitcompile;
++    CompileFunction jitcompile;
++    JitFreeFunction jitfree;
  } PyInterpreterState;
  #endif
  
-diff -r 8667c26e2bec Lib/encodings/aliases.py
---- a/Lib/encodings/aliases.py	Mon Jun 15 09:11:37 2015 -0700
-+++ b/Lib/encodings/aliases.py	Mon Aug 31 14:44:51 2015 -0700
-@@ -1,4 +1,4 @@
--""" Encoding Aliases Support
-+﻿""" Encoding Aliases Support
```

Comment:
Do you know what is going on with all of these minor changes in module docstrings? If not could you strip them from the diff file to minimize patch conflicts and keep the patch focused?